app/views/work_packages/show.html.erb

```diff
@@ -52,7 +52,7 @@ See doc/COPYRIGHT.rdoc for more details.
   <%= render partial: 'show_attributes', locals: { work_package: work_package } %>
 
   <% if work_package.description? %>
-    <hr />
+    <hr>
```

Comment:
The requirements of https://community.openproject.org/work_packages/19073 state to also have an `hr` below the heading.